mcs/class/Mono.Data.Sqlite/Makefile

```diff
@@ -17,6 +17,8 @@ LIB_MCS_FLAGS = /unsafe \
 
 TEST_MCS_FLAGS = $(LIB_MCS_FLAGS) /nowarn:618
 
+TEST_RESOURCE_FILES = test.db
```

Comment:
the test.db is generated further below, but only if the `sqlite3` tool is installed which makes this a bit less than ideal.

what about if we just check in the .db file instead? we already do that here: https://github.com/mono/mono/blob/master/mcs/class/System.Data/Test/System.Data.Common/SqliteTest.db

and we could add an `update-test.db` target which refreshes the .db if we ever make changes to the .sql.